### PR TITLE
refactor(test): remove enzyme from Toaster tests in favor of RTL

### PR DESCRIPTION
**Pull request by ivanhdez28**
re #940

### CODE REVIEW

File: src/components/Toaster/CountdownButton/index.test.tsx [hunk 1]
```diff
@@ -1,19 +1,26 @@
...
-    expect(component).toMatchSnapshot();
+  it('renders component', () => {
+    render(<CountdownButton {...MOCK_PROPS} />);
+    expect(screen.getByTitle('Close')).toBeInTheDocument();
   });
 
+  describe('CountdownButton animation', () => {
+    it('shows a full timer stroke on mount', () => {
+      const { container } = render(<CountdownButton {...MOCK_PROPS} />);
+      const circle = container.querySelector('div > div > svg > circle');
```
Comment (reywright) [inline src/components/Toaster/CountdownButton/index.test.tsx:17]: My perspective is that this is probably against the principles of React Testing Library. We ideally want to test outcomes, not the makeup of the DOM itself. I would try a different approach (or happy to brainstorm with you a bit here).

**Review by reywright — CHANGES_REQUESTED** [review]
Mostly looks good, though one part of the test may be worthy to approach in a different way.

File: src/components/Toaster/CountdownButton/index.test.tsx (hunk 1 above)
Comment (ivanhdez28) [reply src/components/Toaster/CountdownButton/index.test.tsx:17]: true, 100% right what you said, we could probably pair program, there's a few things i couldn't totally grasp about the testing of animations, or maybe i'm even making it a much bigger problem than it is, but still i think could benefit from a pairing session.

Comment (reywright) [reply src/components/Toaster/CountdownButton/index.test.tsx:17]: Totally. Feel free to add something on my calendar for whenever you'd like.

**Review by reywright — APPROVED** [review]
Awesome, thanks.

**Pull request merged**